### ISSUE TITLE
Number of annotations imperceptible by ATs

### ISSUE THREAD

**Issue by klemay** [hypothesis/product-backlog#1108]
This issue was raised by our follow-up accessibility audit. This refers to the tabs in our sidebar:

<img width="509" alt="Screen Shot 2020-05-20 at 12 34 25 PM" src="https://user-images.githubusercontent.com/12087705/82472753-a90a0700-9a96-11ea-90d6-52b87ba79ec4.png">

The number that appears following annotations is not communicated to the screen reader. 

This didn't come up in the last audit - the notes we got were: The number following “Annotation” and “Page Notes” lack intended meaning. Currently a screen reader reports “Annotations one” (for NVDA). Suggest additional screen reader accessible text “Annotations (quantity 1)”

Our developers would like some guidance on what this implementation should look like. It would be helpful for me to know: 

- Is the guidance provided above (say "quantity one" instead of just "one") sufficient for our developers?
- If the answer to the above is no, where does the difficulty lie in fixing this? I see the "Annotations" "Page Notes" and "Orphans" are `<button>`s with a `<span>` that is populated with the count of annotations, page notes, or orphans. Is the trickiness in communicating the contents of a `<span>` that dynamically changes?

**Comment by lyzadanger** [comment]
> Suggest additional screen reader accessible text “Annotations (quantity 1)”

I think we can do this, but is the suggestion to replace the current `aria-label` value with this (i.e. Instead of "Annotations", the `aria-label` would be "Annotations (quantity 1)" or...somewhere else in _addition_ to the `aria-label` value?

**Comment by klemay** [comment]
I will confirm with the auditor- but I believe as long as VoiceOver and NVDA communicate "quantity 1" we are good!
